deploy/crds/keda.sh_triggerauthentications_crd.yaml

```diff
@@ -1,8 +1,18 @@
-apiVersion: apiextensions.k8s.io/v1
+apiVersion: apiextensions.k8s.io/v1beta1
 kind: CustomResourceDefinition
 metadata:
   name: triggerauthentications.keda.sh
 spec:
+  additionalPrinterColumns:
```

Comment:
@zroubalik @ahmelsayed please check whether this is correct

Reply:
That's ok